stats/dummy/collector.go

```diff
@@ -23,13 +23,16 @@ package dummy
 import (
 	"context"
 
+	"github.com/labstack/gommon/log"
```

Comment:
I think `goimports` messed up here. Elsewhere in k6 only `github.com/sirupsen/logrus` is used for logging.

Reply:
😭 that is why `Gopkg.lock` changed. And I wasn't understanding. 😂